consensus/src/user/committee.rs

```diff
@@ -125,14 +126,10 @@ impl Committee {
     }
 
     pub fn total_occurrences(&self, voters: &Cluster<PublicKey>) -> usize {
-        let mut total = 0;
-        for (item_pk, _) in voters.iter() {
-            if let Some(weight) = self.votes_for(item_pk) {
-                total += weight;
-            };
-        }
-
-        total
+        voters
+            .iter()
+            .flat_map(|(voter, _)| self.votes_for(voter))
+            .sum()
```

Comment:
Do you find this more readable than the previous one? It may just be me that finds this more confusing.

Reply:
Actually yes.

However, since readability is subjective, I can revert

Reply:
Then keep it!